core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/kubernetes/WhiskPodBuilder.scala

```diff
@@ -108,7 +108,12 @@ class WhiskPodBuilder(client: NamespacedKubernetesClient, config: KubernetesClie
       .getOrElse(Map.empty)
 
     val diskLimit = config.ephemeralStorage
-      .map(diskConfig => Map("ephemeral-storage" -> new Quantity(diskConfig.limit.toMB + "Mi")))
+      .map(diskConfig =>
+        if (diskConfig.scaleFactor > 0) {
```

Comment:
Can `diskConfig.scaleFactor` be `null`?

Reply:
Didn't think null was a valid value for a Double but, maybe I'm mixing up languages. Will test thanks!

Reply:
based on `KubernetesEphemeralStorageConfig`sig it should not be null, but technically it might be possible; If null was allowed, it should be `Option[Doube]`; I would leave it as is